.travis.yml

```diff
@@ -1,32 +1,15 @@
-language: python
-python:
-  - "3.6"
-addons:
-  postgresql: "9.6"
-
-before_install:
-  - sudo apt-get -qq update
-  - sudo apt-get install -yqq p7zip-full p7zip-rar cpanminus pst-utils gnupg
-  - sudo cpanm --notest Email::Outlook::Message
-  - sudo apt-get install -yqq automake libtool make python zlib1g-dev
-  - git clone https://github.com/hoover/magic-definitions.git
-  - ( cd magic-definitions && ( ./build.sh ) && cp magic.mgc .. )
-  - PATH=`pwd`/magic-definitions/file/bin:$PATH file -m magic.mgc --version
+services:
+  - docker
 
 install:
-  - wget -nc http://archive.apache.org/dist/tika/tika-server-1.17.jar
-  - pip install -r requirements.txt
-  - git clone https://github.com/hoover/testdata.git
-
-env:
-  - SNOOP_TESTDATA=testdata
-
-before_script:
-  - java -jar tika-server-1.17.jar &
-  - sleep 5
+  - docker build . --tag snoop2-testing
+  - git clone https://github.com/hoover/docker-setup.git
+  - ( cd docker-setup && git checkout snoop2-gather-index-into-es && cp docker-compose.override.travis-snoop2.yml docker-compose.override.yml && cp snoop.testing.env snoop.env && cp search.testing.env search.env && cp nginx.testing.env nginx.env && docker-compose up -d )
```

Comment:
I'm ok with having `*.testing.env` in the `docker-setup` repo, but please move them to a subfolder, so they don't spam the root folder. `docker-compose.override.travis-snoop2.yml` should be in the `snoop2` repo though.

Also, before we merge this, let's merge the `snoop2` branch of the `docker-setup` repo. Here is a PR: https://github.com/hoover/docker-setup/pull/3